pyproject.toml

```diff
@@ -52,14 +52,15 @@ module = [
     "trio._abc",
     "trio._core._entry_queue",
     "trio._core._local",
-    "trio._core._unbounded_queue",
     "trio._core._thread_cache",
+    "trio._core._unbounded_queue",
```

Comment:
Unnecessary movement

Reply:
Yes, but that's from sorting them alphabetically. I can change it back of course but I thought there should be at least some order to it.

Reply:
Ah missed that. My bad!

Reply:
(https://github.com/macisamuele/language-formatters-pre-commit-hooks would automatically do this)